python/dgl/dataloading/fixed.py

```diff
@@ -0,0 +1,146 @@
+"""Fixed subgraph sampler."""
+from ..sampling.utils import EidExcluder
+from .base import set_node_lazy_features, set_edge_lazy_features, Sampler
+
+# import non-DGL libraries
+import numpy as np
+import torch
+from collections import defaultdict
+
+class FixedSampler(Sampler):
+    """Subgraph sampler that sets an upper bound on the number of nodes included in 
+    each layer of the sampled subgraph.
+    
+    At each layer, the frontier is randomly subsampled. Rare node types can also be 
+    upsampled by taking the scaled square root of the sampling probabilities.
+
+    It performs node-wise neighbor sampling and returns the subgraph induced by
+    all the sampled nodes.
+
+    Parameters
+    ----------
+    fanouts : list[int] or list[dict[etype, int]]
+        List of neighbors to sample per edge type for each GNN layer, with the i-th
+        element being the fanout for the i-th GNN layer.
+
+        If only a single integer is provided, DGL assumes that every edge type
```

Comment:
Can you follow this example to arrange the bullet points? Please pay attention to the indent.
https://github.com/dmlc/dgl/pull/6998/files#diff-d94a88f81d4e382b3c481a2e2e1b7383ac0b61e39e0d364e8cf457d627356ab8R33

Reply:
Updated to match `python/dgl/graphbolt/sampled_subgraph.py`.